examples/example_seismic.py

```diff
@@ -41,104 +41,78 @@
 
 if __name__ == "__main__":
 
-    #create a seismic dataset from prem:
-    s=burnman.seismic.PREM()
-
-    # specify where we want to evaluate, here we map from pressure to depth
-    #format p = np.arange (starting pressure, ending pressure, pressure step) (in Pa)
-    p = np.arange(1.0e9,360.0e9,5e9)
-    depths = np.array([s.depth(pr) for pr in p])
-    #we could also just specify some depth levels directly like this:
-    #depths = np.arange(35e3,5600e3,100e3)
-    #we could also use the data points where the seismic model is specified:
-    depths = s.internal_depth_list()
-
-    #now evaluate everything at the given depths levels (using interpolation)
-    pressures, density, v_p, v_s, v_phi = s.evaluate_all_at(depths)
+    #List of seismic 1D models 
+    models=[burnman.seismic.PREM(), burnman.seismic.STW105(),burnman.seismic.AK135(),burnman.seismic.IASP91()]
+    colors = ['r','b','m','k']
+    # Variables to plot
+    vars=['pressure','gravity','v_p','v_s','v_phi','density']#,
```

Comment:
vars() is an internal python function which you are hiding here. Another name for this would be better